### PR TITLE
Init-rework draft

### PR DESCRIPTION
**Pull request by StefArma**
Missing where the parameter change server membership to "disabled" and commader switch to "off"

untested yet, might just be wrong

### CODE REVIEW

**Comment by StefArma** [comment]
Tested and working in LAN, need to check on dedicated still. From now on, only an Admin can set campaign restart, membership and commander switch.

Sparker suggested to have a serverfile check like we have for membership: if there is a .txt file with parameters then load those parameters instead the one hardcoded into the mission.
This will help clans who runs advanced medical system and have their own settings
@IrLED could you help with this ?

File: AntistasiOfficial.Altis/initVar.sqf [hunk 1]
```diff
@@ -31,14 +31,9 @@ closeMarkersUpdating = 0;
...
 //get enableRestart from server's parameters in multiplayer
-enableRestart = if (isMultiplayer) then
-{
-	[true, false] select (("AS_enableCampaignReset" call BIS_fnc_getParamValue) == 0)
-}
-else
-{
-	true
-};
+freshstart = if (isMultiplayer) then {[true, false] select (("AS_enableCampaignReset" call BIS_fnc_getParamValue) == 0)} else { true};
```
Comment (IrLED) [inline AntistasiOfficial.Altis/initVar.sqf]: all that can be simplified to 
```
freshstart = !(isMultiplayer) OR {("AS_enableCampaignReset" call BIS_fnc_getParamValue) != 0};
```

File: AntistasiOfficial.Altis/initVar.sqf [hunk 2]
```diff
@@ -31,14 +31,9 @@ closeMarkersUpdating = 0;
...
+commanderswitch =  if (isMultiplayer) then {[true, false] select (("AS_enableSwitchComm" call BIS_fnc_getParamValue) == 0)} else { true};  //missing where to change commanderswitch on if parameters state it.
```
Comment (IrLED) [inline AntistasiOfficial.Altis/initVar.sqf]:
```
freshstart = !(isMultiplayer) OR {("AS_enableSwitchComm" call BIS_fnc_getParamValue) != 0};
```

File: AntistasiOfficial.Altis/initVar.sqf [hunk 3]
```diff
@@ -31,14 +31,9 @@ closeMarkersUpdating = 0;
...
+membership = if (isMultiplayer) then {[true, false] select (("AS_enableServerMember" call BIS_fnc_getParamValue) == 0)} else { true}; //missing where to change membership off if parameters state it.
```
Comment (IrLED) [inline AntistasiOfficial.Altis/initVar.sqf]:
```
membership = !(isMultiplayer) OR {("AS_enableServerMember" call BIS_fnc_getParamValue) != 0};
```

**Pull request merged**